WcaOnRails/spec/requests/api_competitions_spec.rb

```diff
@@ -5,6 +5,43 @@
 RSpec.describe "API Competitions" do
   let(:headers) { { "CONTENT_TYPE" => "application/json" } }
 
+
+  describe "GET #results" do
+    let!(:result) { FactoryBot.create :result, competition: competition }
+
+    it "renders properly" do
+      get api_v0_competition_results_path(result)
+      expect(response).to be_successful
+      json = JSON.parse(response.body)
+      expect(json[0]["id"]).to be >= 0
```

Comment:
Doesn't look like a meaningful check, what about `expect(json[0]["id"]).to eq result.id`.